test/core/test/jest-expect.test.ts

```diff
@@ -133,7 +133,13 @@ describe('jest-expect', () => {
 
     expect('Mohammad').toEqual(expect.stringMatching(/Moh/))
     expect('Mohammad').not.toEqual(expect.stringMatching(/jack/))
-
+    expect({
+      title: '0.1 + 0.2',
+      sum: 0.1 + 0.2,
+    }).toEqual({
+      title: '0.1 + 0.2',
+      sum: expect.closeTo(0.3, 5),
```

Comment:
Not enough tests. What about `not`? What about failing cases?

Reply:
Sure.